atd-toolbox/s3_restore_cr3_pdfs.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/python3
+
+from __future__ import print_function
+
+import re
+import os
+import sys
+import json
+import boto3
+import magic
+import argparse
+import datetime
+import requests
+from operator import attrgetter
+from botocore.config import Config
+
+# https://stackoverflow.com/questions/5574702/how-to-print-to-stderr-in-python
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
```

Comment:
Neat 💯  I need to take your example and make my ETL scripts to print to stderr more often

Reply:
TIL! for what it's worth the `logging` [stream handler](https://docs.python.org/3/library/logging.handlers.html#streamhandler) is probably the more conventional approach.

Reply:
Thanks John - I will check that out and use it in the future.